xml/MAIN.VMDP.xml

```diff
@@ -54,7 +54,40 @@
     to existing operating systems.
    </para>
   </sect2>
-
+  <sect2 xml:id="sec-vmdp-signature">
+   <title>VMDP signature</title>
+   <sect3 xml:id="sec-vmdp-signature-vmdp-win">
+    <title>VMDP-WIN-VERSION</title>
+    <para>
+     VMDP-WIN-VERSION contains the Microsoft WHQL signed sys and cat files for the
```

Comment:
What are 'sys' and 'cat' files? *.sys and *.cat ? then probably enclose in &lt;filename/>

Reply:
this is typical windows systeme file for driver. We can probably replace this detailed information  with "driver" instead.